src/functionalTest/groovy/com/autonomousapps/android/IgnoredVariantSpec.groovy

```diff
@@ -0,0 +1,64 @@
+package com.autonomousapps.android
+
+import com.autonomousapps.android.projects.AllVariantsIgnoredProject
+import com.autonomousapps.android.projects.ReleaseVariantIgnoredProject
+import com.autonomousapps.android.projects.DebugVariantIgnoredProject
+
+import static com.autonomousapps.advice.truth.BuildHealthSubject.buildHealth
+import static com.autonomousapps.utils.Runner.build
+import static com.google.common.truth.Truth.assertAbout
+
+@SuppressWarnings("GroovyAssignabilityCheck")
+final class IgnoredVariantSpec extends AbstractAndroidSpec {
+
+  def "plugin ignore android variants (#gradleVersion AGP #agpVersion ignored debug)"() {
+    given:
+    def project = new DebugVariantIgnoredProject(agpVersion)
+    gradleProject = project.gradleProject
+
+    when:
+    build(gradleVersion, gradleProject.rootDir, 'clean', 'buildHealth', '--no-build-cache')
```

Comment:
why `clean` and `--no-build-cache`?

Reply:
it's copy-paste from `VariantSpec`. I will remove that